<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommLogAnalyzer</class>
 <widget class="QDialog" name="CommLogAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,90">
     <item>
      <widget class="QDateEdit" name="dtLogDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtLogFile"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="twLogs"/>
   </item>
   <item>
    <widget class="QProgressBar" name="pbReader">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
